modules/java_api/build.gradle

```diff
@@ -6,26 +6,33 @@ import org.gradle.internal.os.OperatingSystem
 println 'Operating system: ' + OperatingSystem.current()
 
 def nativesCPP;
-def openvinoVersion = "2022.1"
+def openvinoVersion = "2023.0"
+
+def native_resources = []
+def tbb_dir = System.getenv('TBB_DIR')
+if (tbb_dir) {
+    if (new File(tbb_dir + "/../lib/").exists()) {
+        native_resources.add(tbb_dir + "/../lib/");
+    } else {
+        native_resources.add(tbb_dir + "/../../");
+    }
+}
 
 if (OperatingSystem.current().isMacOsX()) {
     nativesCPP = 'macosx-x86_64'
+    native_resources.add(System.getenv('INTEL_OPENVINO_DIR') + "/runtime/lib/intel64/Release");
```

Comment:
what's about ARM64?

Reply:
Done

Reply:
I suppose `intel64` should be replaced as well to arm64 / aarch64 depending on Linux / MacOS / Windows

Reply:
Added this option too, thanks